wordfreq_builder/wordfreq_builder/ninja.py

```diff
@@ -236,27 +236,51 @@ def jieba_deps(dirname_in, languages):
     return lines
 
 
+def reddit_base_filename(path):
+    """
+    Get the base name of a Reddit input file, without its path or extension.
+    """
+    return path.stem
```

Comment:
... also, this is a one-line function that is used in one place, so it doesn't really need to be a function.